chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -727,6 +728,8 @@ search.complianceFilter.unavailable=Compliance and non-conformity filtering is u
 search.hasHadSurveillance.invalid=Parameter value '%s' for hasHadSurveillance is invalid. It must be either 'true' or 'false'.
 search.hasHadComplianceActivity.invalid=Parameter value '%s' for hasHadComplianceActivity is invalid. It must be either 'true' or 'false'.
 search.nonconformitySearchOption.invalid=No non-conformity search option matches '%s'. Values must be one of %s.
+search.rwt.missingSearchOperator=Multiple RWT search options were found without a search operator (AND/OR). A search operator is required.
```

Comment:
Is this error still used? I thought we defaulted to `OR` if this operator isn't found

Reply:
It's still used in the POST version of the endpoint. That one doesn't make use of the same default parameter values nor does it have the same notes about that in Swagger. If we need to refine that endpoint's behavior and documentation could/should that be a different ticket or do you think it needs done in this one?